tests/by-util/test_mv.rs

```diff
@@ -1791,3 +1793,16 @@ fn test_mv_error_msg_with_multiple_sources_that_does_not_exist() {
         .stderr_contains("mv: cannot stat 'a': No such file or directory")
         .stderr_contains("mv: cannot stat 'b/': No such file or directory");
 }
+
+#[cfg(not(windows))]
+#[ignore = "requires access to a different filesystem"]
+#[test]
+fn test_special_file_different_filesystem() {
+    let (at, mut ucmd) = at_and_ucmd!();
```

Comment:
https://github.com/uutils/coreutils/pull/7249
might be useful :)

Reply:
I just now tried to use the new `TestScenario::mount_temp_fs()` but I couldn't figure out how to get this test working with it. I just left a TODO comment mentioning it.

Reply:
@matrixhead any idea ? :)